bin/pygrb/pycbc_pygrb_minifollowups

```diff
@@ -0,0 +1,205 @@
+
+# Create a FileList that will contain all output files
+layouts = []
+
+# Read the file with the triggers/injections to follow up
+logging.info('Reading list of triggers/injections to followup')
+fp = h5py.File(args.followups_file, "r")
+
+# Initialize a wiki table and add the column headers
+if args.wiki_file:
+    wiki_file = os.path.join(args.output_dir, args.wiki_file)
+    add_wiki_row(wiki_file, fp.keys())
+
+# Trigger time
+seg_dir = workflow.cp.get('workflow', 'segment-dir')
+trig_time = get_grb_time(seg_dir)
+
+# Establish the number of follow-ups to perform
+num_events = int(workflow.cp.get_opt_tags('workflow-minifollowups',
+                                          'num-events', ''))
+num_events = min(num_events, len(fp['BestNR'][:]))
+
+# Determine ifos used in the analysis
+trig_file = workflow.cp.get('workflow', 'trig-file')
+ifos = extract_ifos(trig_file)
+num_ifos = len(ifos)
+
+# (Loudest) off/on-source events are on time-slid data so the
+# try will succeed, as it finds the time shift columns.
+is_injection_followup = True
+try:
+    time_shift = fp[ifos[0]+' time shift (s)'][0]
+    is_injection_followup = False
+except:
+    pass
+
+
+# Loop over triggers/injections to be followed up
+for num_event in range(num_events):
+    files = wf.FileList([])
+    logging.info('Processing event: %s', num_event)
+    gps_time = fp['GPS time'][num_event]
+    gps_time = gps_time.astype(np.float)
+    if wiki_file:
+        row = []
+        for key in fp.keys():
+            row.append(fp[key][num_event])
+        add_wiki_row(wiki_file, row)
+    # Handle off/on-source (loudest) triggers follow-up (which are on slid data)
+    if not is_injection_followup:
+        for ifo in ifos:
+            time_shift = fp[ifo+' time shift (s)'][num_event]
+            ifo_time = gps_time - time_shift
+            files += make_timeseries_plot(workflow, 'single', gps_time, ifo_time,
+                                          args.output_dir, ifo=ifo,
+                                          tags=args.tags + [str(num_event)])
+            files += mini.make_qscan_plot(workflow, ifo, ifo_time,
+                                          args.output_dir,
+                                          tags=args.tags + [str(num_event)])
+    # Handle injections (which are on unslid data)
+    else:
+        for snr_type in ['reweighted', 'coherent']:
+            files += make_timeseries_plot(workflow, snr_type, gps_time, gps_time,
+                                          args.output_dir, ifo=None,
+                                          tags=args.tags + [str(num_event)])
+        for ifo in ifos:
+            files += mini.make_qscan_plot(workflow, ifo, gps_time,
+                                          args.output_dir,
+                                          tags=args.tags + [str(num_event)])
+
+    layouts += list(layout.grouper(files, 2))
+
+workflow.save(filename=args.output_file, output_map_path=args.output_map)
```

Comment:
And finally here just have `workflow.save()` 

The example for this is https://github.com/gwastro/pycbc/blob/master/bin/minifollowups/pycbc_foreground_minifollowup